reputation.php

```diff
@@ -532,12 +532,14 @@
 	// Verify incoming POST request
 	verify_post_check($mybb->get_input('my_post_key'));
 
+	$rid = $mybb->get_input('rid', MyBB::INPUT_INT);
+
```

Comment:
<- And add another one here named (possibly) `reputation_delete_start` ;)

Reply:
Ok, will change in 8 hours when I get home

Reply:
Done, good suggestion!